justfile

```diff
@@ -119,7 +119,7 @@ coverage: init-llvm-cov
 
     cargo +nightly llvm-cov test --workspace --no-report --all-targets --all-features
     cargo +nightly llvm-cov test --workspace --no-report --doc --doctests --all-features
-    cargo +nightly llvm-cov report --codecov --output-path {{join(COV_OUTPUT_DIR, "hickory-dns-coverage.json")}}
+    cargo +nightly llvm-cov report --doctests --codecov --output-path {{join(COV_OUTPUT_DIR, "hickory-dns-coverage.json")}}
```

Comment:
`--help` reports this as unstable... not sure if it's a good idea to put it in there?

(I would suspect our doctests wouldn't generate much additional coverage, anyway.)

Reply:
We are already using this flag when running tests above, and it is also unstable in that context. Currently, we run doctests, but don't include doctest coverage in reports. With this change, coverage from those tests will be included.